kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/CreateCheckpointIterator.java

```diff
@@ -352,6 +365,30 @@ private void processTxn(ColumnVector txnVector, boolean[] selectionVectorBuffer)
     }
   }
 
+  private void processDomainMetadata(
```

Comment:
what does `process` mean? javadocs please

Reply:
The name is to follow the naming convention used across similar functions (e.g., processMetadata, processTxn, etc.). I’ve added a Javadoc to explain its purpose and behavior.